storehaus-hbase/src/main/scala/com/twitter/storehaus/asynchbase/AsyncHBaseStore.scala

```diff
@@ -20,7 +20,6 @@ import com.twitter.util.{Future, FuturePool}
 import java.util.concurrent.Executors
 import org.hbase.async.{DeleteRequest, PutRequest, GetRequest, HBaseClient}
 import com.twitter.bijection.{Injection, Codec, Conversion}
-import scala.Some
 import Conversion._
 import scala.collection.JavaConverters._
 
```

Comment:
maybe not reuse the same name there, maybe call the above proposal something different.